stdlib/src/builtin/int_literal.mojo

```diff
@@ -269,6 +269,47 @@ struct IntLiteral(
         """
         return self
 
+    @always_inline("nodebug")
+    fn __divmod__(self, rhs: Self) -> Tuple[Self, Self]:
+        """Return the quotient and remainder of the division of self by rhs.
+
+        Args:
+            rhs: The value to divide on.
+
+        Returns:
+            The quotient and remainder of the division.
+        """
+        var quotient: Self = self.__floordiv__(rhs)
+        var remainder: Self = self - (quotient * rhs)
+        return quotient, remainder
+
+    @always_inline("nodebug")
+    fn __round__(self, ndigits: IntLiteral) -> Self:
```

Comment:
Both here and in the implementation for `FloatLiteral`: Given that `ndigits` is realistically not bigger than what we can store in `Int`, `IntLiteral` is not necessary. This might cause some calculations to happen in finite precision (i.e. return `Int` instead of `IntLiteral`); you can add some special methods for these mixed type computation.

Reply:
Done: https://github.com/msaelices/mojo/commit/9de5acca6db1dad73f160a084687cdfaba9f5ed4